src/erlazure_insights.erl

```diff
@@ -0,0 +1,132 @@
+-module(erlazure_insights).
+
+-export([
+        get_events/4,
+        get_time_offset_as_timestamp/2,
+        get_events_all/4,
+        follow_token/3
```

Comment:
The reason for exporting this function was that in the Acceptence Criteria on the Rally US there is
```
there is function call to fetch events with token
```

However I can remove the export line for this function without any side effects